enzy_htp/structure/structure.py

```diff
@@ -204,6 +204,32 @@ def _legal_new_chain_names(self) -> List[str]:
         result = list(filter(lambda s: s not in self.chain_names, result))
         return result
 
+    @property
+    def aa_chains(self) -> List[Chain]:
+        result = list()
+        for chain in self.chains:
+            if not len(chain.residues):
+                continue
+            
+            if chain.residues[0].is_canonical():
+                result.append( chain )
+
+        return result
```

Comment:
Is this the same as .polypeptide? Sorry for missing this is the 1st round of comment

Reply:
Yes, same idea I think.